docs/dev/api/insights.md

```diff
@@ -3152,19 +3815,163 @@ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
 
 ---
 
-### Get Coverage CSV
+### Get Coverage from All Sources
 
-<details><summary><span className="api get">GET</span> <code>/v2/insights/&#123;source&#125;/coverage/&#123;coverage_field&#125;/csv</code></summary>
+<details><summary><span className="api get">GET</span> <code>/insights/v2/coverage/&#123;coverage_field&#125;</code></summary>
 <p/>
 
-Return information about tests coverage for the specified `coverage_field` in a CSV format.
+Return information, from all sources, about test coverage for the specified <code>coverage_field</code>.
 
 #### Parameters
 
 <table id="table-api">
   <tbody>
     <tr>
-     <td><code>source</code></td>
+     <td><code>coverage_field</code></td>
+       <td><p><small>| PATH | REQUIRED | STRING |</small></p><p>Available values are:<code>device</code>, <code>browser</code>, <code>os</code>.</p><p>
```

Comment:
When there is more than one available value, we should use a bulleted list so the description is easier to read.
```suggestion
       <td><p><small>| PATH | REQUIRED | STRING |</small></p><p>Available values are:<ul><li><code>device</code></li><li><code>browser</code></li><li><code>os</code></li></ul>.</p><p>
```